ethcore/evm/src/factory.rs

```diff
@@ -101,24 +74,7 @@ fn test_create_vm() {
 /// Create tests by injecting different VM factories
 #[macro_export]
 macro_rules! evm_test(
-	(ignorejit => $name_test: ident: $name_jit: ident, $name_int: ident) => {
-		#[test]
-		#[ignore]
-		#[cfg(feature = "jit")]
-		fn $name_jit() {
-			$name_test(Factory::new(VMType::Jit, 1024 * 32));
-		}
-		#[test]
-		fn $name_int() {
-			$name_test(Factory::new(VMType::Interpreter, 1024 * 32));
-		}
-	};
-	($name_test: ident: $name_jit: ident, $name_int: ident) => {
-		#[test]
-		#[cfg(feature = "jit")]
-		fn $name_jit() {
-			$name_test(Factory::new(VMType::Jit, 1024 * 32));
-		}
+	($name_test: ident: $name_int: ident) => {
```

Comment:
This could be further removed (inlined), since now we only create one test.

Reply:
should I just remove this macro completely? Cause this is the only way to 'inline' this test further. Please note, that this macro creates function with name `$name_int` which calls `$name_test`.

Reply:
Yeah, that's what I thought.